packages/marko-web-html-sitemap/templates/date-list.marko

```diff
@@ -0,0 +1,60 @@
+$ const { config, GAM } = out.global;
+$ const { nodes, displayType, month, year, alias } = input;
+
+$ const type = "sitemap";
+$ const pageNode = {
+  title: 'Sitemap',
+};
+$ switch (displayType) {
+  case "day":
+      pageNode.description = `Select from the list of days in ${month} ${year}`;
+      break
+  case "month":
+      pageNode.description = `Select from the list of months in ${year}`;
+      break
+  default:
+    pageNode.description = `Select from the list of years`;
+};
+
+$ const nameOfMonth = [
```

Comment:
Use a library to get the localized version of the month name. On mundo this should be `junio` rather than `July`, etc. Or use the translation utility to get the name of the month (`MONTH_1_LABEL` or something)

Reply:
@solocommand I just used moment to format this.  will figure out lang support on second PR